ultimatejobweb/models.py

```diff
@@ -1,17 +1,17 @@
 from django.db import models
-from django.contrib.auth.models import User
-
-
-class user(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    profession = models.CharField(max_length=128)
 
 
 class company(models.Model):
```

Comment:
python naming standards are for the class to have capital name (CamelCase)
and for variables/attributes to be lower_case with underscores separating the variable name.
You did it opposite.